steps/unzip_submission.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
```

Comment:
This script can be part of the `docker.synapse.org/syn53065762/validate_score:v4` docker model too, if you wanted.

Reply:
Thanks I figured I put it in here so I had a reference in this repo about what I did to handle the need for a config file to be created for each submission.